Sprint-2/debug/recipe.js

```diff
@@ -11,5 +13,4 @@ const recipe = {
 };
 
 console.log(`${recipe.title} serves ${recipe.serves}
```

Comment:
The goal is to print each ingredient on a new line. How do you feel that can be achieved?

Reply:
> The goal is to print each ingredient on a new line. How do you feel that can be achieved?

Yes, since ingredients is an array, I can use a for ...of loop to print each item on a new line